src/rollup/types.d.ts

```diff
@@ -0,0 +1,343 @@
+import { EventEmitter } from 'events';
+
+// ast node types just any, unless acorn public dependency added
+export type Program = any;
+export type Node = any;
+
+export interface IdMap {
+	[key: string]: string;
+}
+
+export interface RollupError {
+	message: string;
+	code?: string;
+	name?: string;
+	url?: string;
+	id?: string;
+	loc?: {
+		file?: string;
+		line: number;
+		column: number;
+	};
+	stack?: string;
+	frame?: string;
+	pos?: number;
+	plugin?: string;
+	pluginCode?: string;
+}
+
+export type VERSION = string;
```

Comment:
This doesn't look right, AFAIK version return a string value and thus it should be a `const` and not `type`

The usage I believe it should be;

```ts
import { VERSION } from 'rollup';
const rollupVersion = VERSION:
```

Reply:
Thanks